src/app/authentication/application/token.service.ts

```diff
@@ -1,21 +1,27 @@
+import axios, { AxiosInstance } from 'axios';
+import { inject, injectable } from 'inversify';
 import { sign, verify } from 'jsonwebtoken';
 import { ConfigurationService } from '../../core/model/configuration.model';
+import { TokenType } from '../domain/enums';
 import {
-    TokenService,
-    TokenPayload,
     AdminTokenPayload,
-    TokenRepository
+    TokenPayload,
+    TokenRepository,
+    TokenService
 } from '../model/token.model';
 import { User, UserToken } from '../model/user.model';
-import { TokenType } from '../domain/enums';
-import { injectable, inject } from 'inversify';
 import { APPLICATION_TYPES } from './../../application.types';
-
 @injectable()
 export class DefaultTokenService implements TokenService {
     private expirationTime = 60 * 60 * 24;
     private adminExpirationTime = 60 * 60 * 24 * 7;
     private jwtSecret: string;
+    protected redirectionTarget: AxiosInstance = axios.create({
+
+        //baseURL: "https://mibi-portal.bfr.bund.de" + API_ROUTE.V2
+        baseURL: "https://epilab-dev.bfr.berlin/admin/parse"
+    });
```

Comment:
Is the epilab-dev url for testing purposes now?

Reply:
Yes.  Makes things easier.  Will be fixed in Ticket #420